### DISCUSSION TITLE
Error while integrating w3m in my react native app

### DISCUSSION THREAD

**Discussion by ishkash69** [WalletConnect/walletconnect-monorepo#3985] [Developer Support]
hi @ALL please help me out with this issue when i try to integrate new version of w3m in my react native app
this is the issue i got after the integration to my react native app.

<img width="428" alt="Screen Shot 2023-11-27 at 11 19 04 AM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/107914962/1e2cc8cf-8928-434f-bc41-6f0e5744ef7a">

**Comment by glitch-txs** [comment]
hi, could you open an issue in React Native repository? please add a minimal reproducible example of the issue so we can test the code, Thanks!
